lib/tests/formulae_dependents.rb

```diff
@@ -22,19 +24,29 @@ def dependent_formulae!(formula_name, args:)
 
         test_header(:FormulaeDependents, method: "dependent_formulae!(#{formula_name})")
 
+        formula = Formulary.factory(formula_name)
+
+        source_dependents, bottled_dependents, testable_dependents =
+          dependents_for_formula(formula, formula_name, args: args)
+
+        return if source_dependents.blank? &&
+                  bottled_dependents.blank? &&
+                  testable_dependents.blank?
+
         # Install formula dependencies. These will have been uninstalled after building.
         test "brew", "install", "--only-dependencies", formula_name,
              env: { "HOMEBREW_DEVELOPER" => nil }
         return if steps.last.failed?
 
         # Restore etc/var files that may have been nuked in the build stage.
-        test "brew", "postinstall", formula_name
+        formula_dependencies = Utils.safe_popen_read("brew", "deps", formula_name).split("\n")
```

Comment:
Are we calling `brew deps` anywhere else? If so, worth caching?

Reply:
I'll have a look, but I don't think so it's used elsewhere.